src/components/FormQuestion.js

```diff
@@ -4,14 +4,17 @@ import { View, Text } from 'react-native';
 import CheckboxGroup from './CheckboxGroup';
 import RadioGroup from './RadioGroup.js';
 import DropDown from './DropDown';
+import TextField from './Login/TextFields.js';
+// import SectionHeader from './SectionHeader.js';
```

Comment:
FormQuestion only accepts 'input-like' types.  SectionHeader would need to be a separate component.